pom.xml

```diff
@@ -72,11 +72,17 @@
         </dependency>
         <dependency>
             <groupId>org.slf4j</groupId>
-            <artifactId>slf4j-log4j12</artifactId>
+            <artifactId>slf4j-reload4j</artifactId>
```

Comment:
**SideNote**: As gateleen is a library, it should depend solely on "*slf4j-api*" but nothing else. Especially no logger implementations as those potentially mess up with all dependents of gateleen. The only module in gateleen permitted to pull in a logger impl should be the playground module, as it is an application using the gateleen library. I wonder why this dependency is even here.

Reply:
This is not gateleen but vertx-redisques